server/src/generate_once.ts

```diff
@@ -10,41 +12,58 @@ program.parse();
 const options = program.opts();
 
 (async () => {
+    const CACHE_PATH = path.resolve(__dirname, "cache.json");
```

Comment:
```suggestion
    const CACHE_PATH = path.resolve(__dirname, "..", "cache.json");
```
do you think it should be in the same dir with `package.json`?

Reply:
i think so